sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -202,7 +202,7 @@ def s3_event_handler(event):
     #     "extension": self.extension,
     #     "ingestion_date": date_object,
     # }
-    metadata_params = science_file.get_file_metadata_params()
+    metadata_params = science_file.extract_filename_components(filename)
```

Comment:
I think this is a static method and should take as input the science_file you created?
```suggestion
    metadata_params = SciencFilePath.extract_filename_components(science_file)
```

Should we change "metadata_params" to "file_params" or something more descriptive? I'm not sure it is metadata. Can save this for a follow-up small PR.

Reply:
I can change that to be `file_params`.

Reply:
It should actually probably be `ScienceFilePath.extract_filename_components(filename)` and you remove the `science_file` variable entirely. Since you're just extracting the filename components, you don't need to use the rest of the class.

Reply:
I thought of doing that but was hesitant. will change to do that.